src/lecturer/components/studentListView/StudentListHead.tsx

```diff
@@ -27,9 +28,31 @@ export function StudentListHead(props: StudentListHeadProps) {
         onRequestSort(event, property);
     };
 
+    const theme = useTheme();
+
+    const classes = makeStyles({
+        root: {
+            maxWidth: "600px",
+            margin: "15px auto"
+        },
+        head: {
+            padding: "20px 15px",
+            textAlign: "left",
+            fontWeight: 500,
+            color: "#fff",
+            
+            backgroundColor: theme.palette.primary.main,
+            "& span": {
```

Comment:
What does it mean? ("& span")?

Reply:
It's selector pointing to upper selector

For example:
head: {
    width: "200px",
    "& td": {
        color: "red"
    }
}

in CSS means:
.head{
    width: 200px;
}
.head td{
    color: red;
}

Reply:
thanks